Gemfile

```diff
@@ -1,7 +0,0 @@
-source 'https://rubygems.org'
```

Comment:
Sorry, my mistake, I had to delete it in order to run Jekyll over OSX, but it shouldn't commit this change. Will revert this deletion.